runtime/lua/vim/lsp/util.lua

```diff
@@ -843,36 +853,47 @@ function M.convert_signature_help_to_markdown_lines(signature_help, ft, triggers
         documentation?: string | MarkupContent;
       }
       --]=]
```

Comment:
would the above comment benefit from a "before => after" example to show what's going on here?

Reply:
I'm not sure I understand. What exactly would you like me to clarify further?